src/MLOps.NET/MLLifeCycleManager.cs

```diff
@@ -71,6 +72,20 @@ public async Task UploadModelAsync(Guid runId, string filePath)
             await ModelRepository.UploadModelAsync(runId, filePath);
         }
 
+        ///<inheritdoc/>
+        public IRun GetBestRun(Guid experimentId, string metricName)
+        {
+            EnsureStorageProviderConfigured();
+            var allRuns = MetaDataStore.GetRuns(experimentId);
+            // Flattening the metrics for all the runs for a given experiment into one list and finding the best among them.
```

Comment:
Your code is self-explanatory so no comment needed :) In general I would like to try to keep comments to summary XML comments for intellisense.

Reply:
Thanks :) removed...